tests/typescript/argon2-weak-type-typescript-test.yml

```
@@ -0,0 +1,9 @@
+id: argon2-weak-type-typescript
+valid:
+  - |
+    await argon2.hash('password', {type: argon2.argon2id})
+    await argon2.hash('password', {})
+invalid:
+  - |
+    await argon2.hash('password', {type: argon2.argon2d})
+    await argon2.hash('password', {type: argon2.argon2i})
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)